packages/boxel/addon/styles/variables.css

```diff
@@ -119,6 +119,10 @@
 
   /* Fix: Other colors used in app, potentially not in color palette: */
 
+  /* z-index layers */
+  --boxel-layer-modal-default: 15;
+  --boxel-layer-modal-urgent: 20;
+
```

Comment:
Should this go below the following colours, so the comment isn’t mixed up?

Reply:
I think keeping the variables visible is better than adding it below the colours to keep consistency, it feels like someone might scroll along and see the "less-used colors" and not realize there are z-indices around.

Reply:
Will move it up. Thanks!